Sources/App/app.swift

```diff
@@ -9,3 +9,20 @@ public func app(_ env: Environment) throws -> Application {
     let app = try Application(config: config, environment: env, services: services)
     return app
 }
+
+extension Environment {
+    enum Key: String {
+        case SLACK_TOKEN
+        case SLACK_CHANNEL
+        case GITHUB_USERNAME
+        case GITHUB_TOKEN
+        case CIRCLECI_TOKEN
+        case JIRA_HOST
+        case JIRA_USERNAME
+        case JIRA_TOKEN
+    }
```

Comment:
Btw I think all those would be "swiftier" if they followed the Swift naming convention for cases, even if that means explicitly specifying the upper-case rawValue

```swift
case jiraToken = "JIRA_TOKEN"
```

Reply:
Actually it should even be a `struct Key`, not an `enum`, as it doesn't make sense semantically for it to be an `enum` (we're not gonna `switch` on it, and it could make sense for another file to use `extension Key` to add more constants to it if they need some, etc...)